web/app/dashboard/dashboard.component.ts

```diff
@@ -14,12 +16,24 @@ export class DashboardComponent implements OnInit {
       ['name', 'latestBuild', 'repo', 'lane'];
   isLoading = true;
   projects: ProjectSummary[];
-  constructor(private readonly dataService: DataService) {}
+
+  constructor(
+      private readonly dataService: DataService,
+      private readonly dialog: MatDialog) {}
 
   ngOnInit() {
     this.dataService.getProjects().subscribe((projects) => {
       this.projects = projects;
       this.isLoading = false;
     });
   }
+
+  openAddProjectDialog() {
+    const dialogRef =
+        this.dialog.open(AddProjectDialogComponent, {width: '250px', data: {}});
```

Comment:
I feel like the default line width could be expanded to 120 before wrapping like this, but I'm not super familiar with web coding styling.

Reply:
Yeah something i wanted to look into was encapsulating Google codestyle requirements in tslint. So we can just autoformat. But i'm punting that until later on. Hope that's OK.

Reply:
Yeah, sounds good. We can ignore style stuff and then autoformat later 🌮